alerter/AlerterUnitTest.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace alerter
+{
+  class AlerterUnitTest
+  {
+    static void Main(string[] args)
+    {
+      Alerter.AlertInCelsius(303.6f, "Production");
+      Debug.Assert(Alerter.alertFailureCount == 0);
+      
+      Alerter.AlertInCelsius(400.5f, "Test");

```

Comment:
sufficient boundary conditions and beyond range needs to be added to make robust testing.

Reply:
Boundary values are added for the robust testing